src/TiledArray/conversions/concat.h

```diff
@@ -92,9 +92,8 @@ DistArray<Tile, Policy> concat(
   DistArray<Tile, Policy> result(*target_world, tr);
   const auto annot = detail::dummy_annotation(r);
   for (auto i = 0ul; i != arrays.size(); ++i) {
-    result.make_tsrexpr(annot).block(tile_begin_end[i].first,
-                                     tile_begin_end[i].second) =
-        arrays[i].make_tsrexpr(annot);
+    result(annot).block(tile_begin_end[i].first, tile_begin_end[i].second) =
```

Comment:
The seems to undo the changes introduced in https://github.com/ValeevGroup/tiledarray/pull/461 ?